config/boot.rb

```diff
@@ -1,4 +1,4 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
-require 'bootsnap/setup' # Speed up boot time by caching expensive operations.
+require 'bootsnap/setup' if ENV['RAILS_ENV'] == 'production' # Speed up boot time by caching expensive operations.
```

Comment:
You shouldnt access the env var directly really - cos that is using internal knowledge of rails - instead use the method that rails provides

```
Rails.env.production?
```

unless for some reason this is not accessible this early in the boot process

Reply:
Good point, thanks! Fixed in 910d530

Reply:
I lied, this kills the crab. Will have to revert.